src/create/document.ts

```diff
@@ -45,8 +53,35 @@ export interface ZodOpenApiResponsesObject
     | oas31.ReferenceObject;
 }
 
+export type AnyObjectZodType = ZodType<object, any, any>;
+
+export function getZodObject(schema: AnyObjectZodType): AnyZodObject {
```

Comment:
```suggestion
export const getZodObject = (schema: AnyObjectZodType): AnyZodObject => {
```

nit: just gonna keep it consistent across the repo